github/repos_pages.go

```diff
@@ -93,6 +93,25 @@ func (s *RepositoriesService) GetLatestPagesBuild(owner string, repo string) (*P
 	return build, resp, err
 }
 
+// GetPageBuild fetches the specific build information for a GitHub pages site.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/pages/#list-a-specific-pages-build
+func (s *RepositoriesService) GetPageBuild(owner string, repo string, ID int) (*PagesBuild, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pages/builds/%v", owner, repo, ID)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
I can't get any of the Pages Preview API calls to work with or without that header. 
```
curl "https://api.github.com/repos/github-archive/developer.github.com/pages/builds" -H "Accept: application/vnd.github.mister-fantastic-preview" -i
```

```
HTTP/1.1 404 Not Found
Server: GitHub.com
Date: Tue, 11 Oct 2016 21:48:19 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 87
Status: 404 Not Found
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4936
X-RateLimit-Reset: 1476223898
X-OAuth-Scopes: repo
X-Accepted-OAuth-Scopes: 
X-GitHub-Media-Type: github.mister-fantastic-preview
Access-Control-Expose-Headers: ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Content-Security-Policy: default-src 'none'
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 68842252:19A5:7E49F4B:57FD5E23

{
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3"
}
```